### PR TITLE
MC-12144: Add API docs for verified domains, security settings

### PR DESCRIPTION
**Pull request by lamminade**
#### Changes made
- Added verified domains and security settings to the organizations page in administration

#### Related PRs
- [UI](https://github.com/cloudops/cloudmc-ui/pull/971)
- [core](https://github.com/cloudops/cloudmc-core/pull/884)

### CODE REVIEW

File: source/includes/administration/_organizations.md [hunk 1]
```diff
@@ -233,3 +233,209 @@ curl -X DELETE "https://cloudmc_endpoint/v1/organizations/03bc22bd-adc4-46b8-988
...
+
+<!-------------------- CREATE VERIFIED DOMAIN -------------------->
+### Create verified domain
+
+`POST /organizations/:organization_id/verified_domains`
+
+Creates a new verified domain in the specified organization.
+
+```shell
+# Create an organization
```
Comment (franzpgarcia) [inline source/includes/administration/_organizations.md]: create verified domain

File: source/includes/administration/_organizations.md [hunk 2]
```diff
@@ -233,3 +233,209 @@ curl -X DELETE "https://cloudmc_endpoint/v1/organizations/03bc22bd-adc4-46b8-988
...
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "lastCheckedDate": "2020-07-31T19:14:36Z",
+      "createdDate": "2020-07-31T19:12:55Z",
+      "domain": "www.aGoodFakeDomain.com",
```
Comment (franzpgarcia) [inline source/includes/administration/_organizations.md]: i'll change it to never return `www.` and `http://`. Could you remove please 🙏

**Review by franzpgarcia — APPROVED** [review]

File: source/includes/administration/_organizations.md (hunk 2 above)
Comment (lamminade) [reply source/includes/administration/_organizations.md]: 9a27d69

File: source/includes/administration/_organizations.md (hunk 1 above)
Comment (lamminade) [reply source/includes/administration/_organizations.md]: 9a27d69

**Review by evanluc — APPROVED** [review]

**Pull request merged**